apps/desktop-wallet/src/features/historicChart/useHistoricData.tsx

```diff
@@ -109,17 +110,17 @@ const useHistoricData = () => {
         },
         {} as Record<AddressHash, Record<Timestamp, Amount> | undefined>
       ),
-      isPending: results.some(({ isPending }) => isPending),
+      isLoading: results.some(({ isLoading }) => isLoading),
       hasHistoricBalances: results.some(({ data }) => data?.amountHistory?.length)
     })
   })
 
   return {
     alphBalanceHistoryPerAddress,
-    isPendingHistoricalAlphBalances,
-    isPendingAlphPriceHistory,
+    isLoadingHistoricalAlphBalances,
+    isLoadingAlphPriceHistory,
     alphPriceHistory,
```

Comment:
Should we systematically structure the output data in an object like `data: {....}`?
Also, is it useful to keep 2 different `isLoading` states?

Right now I think that it'd make things clearer if data hooks systematically return an object structured like `{data: {}, isLoading: boolean}`. WDYT?